osi_ippay_payment/models/account_payment.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountRegisterPayments(models.TransientModel):
```

Comment:
Should this rather be in a `wizard/` subdirectory?

Reply:
or `wizards` according to OCA guidelines (https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id9)